Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -613,10 +647,53 @@ private async void CheckForIdlePlayback(object state)
                 playingSessions = Sessions.Where(i => i.NowPlayingItem is not null)
                     .ToList();
             }
+            else
+            {
+                StopIdleCheckTimer();
+            }
+        }
 
+        private async void CheckForInactiveSteams(object state)
+        {
+            var pausedSessions = Sessions.Where(i =>
+                    i.NowPlayingItem is not null
+                    && i.PlayState.IsPaused
+                    && i.LastPausedDate is not null)
+                .ToList();
+
+            if (pausedSessions.Count > 0)
+            {
+                var inactiveSessions = pausedSessions.Where(i => (DateTime.UtcNow - i.LastPausedDate).Value.TotalMinutes > _config.Configuration.InactiveSessionThreshold).ToList();
+
+                foreach (var session in inactiveSessions)
+                {
+                    _logger.LogDebug("Session {0} has been inactive for {1} minutes. Stopping it.", session.Id, _config.Configuration.InactiveSessionThreshold);
```

Comment:
Named placeholders are preferred, at least for new code.

```suggestion
                    _logger.LogDebug("Session {Session} has been inactive for {InactiveTime} minutes. Stopping it.", session.Id, _config.Configuration.InactiveSessionThreshold);
```